source/scripting_v3/GTA.NaturalMotion/EuphoriaBase.cs

```diff
@@ -83,6 +102,83 @@ public void SendTo(Ped target, int duration)
 			SendTo(target);
 		}
 
+		/// <summary>
+		/// Sends message this NaturalMotion behavior on the <see cref="Ped"/>, but don't start a new ragdoll task so the <see cref="Ped"/> can have multiple NM behaviors.
+		/// You'll need to start a ragdoll task with <see cref="Ped.Ragdoll(int, RagdollType)"/> before the <see cref="Ped"/> can recieve this <see cref="Message"/>.
+		/// </summary>
+		/// <remarks>
+		/// Although <see cref="SendTo(Ped)"/> wouldn't create a new ragdoll task if the <see cref="Ped"/> runs a script ragdoll tasks in most cases,
+		/// this method guarantees it sends the message if the <see cref="Ped"/> can receive NM messages but doesn't create a new ragdoll task.
+		/// </remarks>
+		/// <param name="target">The <see cref="Ped"/> to send the <see cref="Message"/> to.</param>
+		/// <param name="startBehavior">Specifies whether this message should start from the beginning.</param>
+		/// <param name="ifNMScriptControlRunning">
+		/// <para>Specifies whether the messages should be sent to the only if they <see cref="Ped"/> has a <c>CTaskNMScriptControl</c> running, which can be started with <see cref="Ped.Ragdoll(int, RagdollType)"/>.</para>
+		/// <para>
+		/// If you set this parameter to <see langword="false"/> and you need to set the <see cref="Ped"/> from a ragdoll state to an animated state after a certain timeout (in game time),
+		/// you should call <see cref="Ped.CancelRagdoll()"/> on them after a certain timeout if they are alive and <see cref="Ped.IsRagdoll"/> returns <see langword="true"/>.
+		/// </para>
+		/// </param>
+		public void SendToNoNewRagdollTask(Ped target, bool startBehavior = true, bool ifNMScriptControlRunning = true)
```

Comment:
We should be able to make due with the existing `SendTo`, rather than introducing an additional method. It already checks whether the ped is in ragdoll (may have to expand that to use the new `IsRunningRagdollTask`) and only starts a new ragdoll task if not so.
To make it easier I'd also change it so that `SendTo` no longer sets the `start` argument in any case, rather it should depend on that being set in the message already (or not). That way we can specify that behavior in `CustomHelper.Start()` (set `start` and afterwards remove it again) and `CustomHelper.Update()` (don't set `start`).

Reply:
No need to use `IsRunningRagdollTask` before sending NM messages since it returns `true` when the ped is in writhe or getting up btw, where they don't receive NM messages at all.

Reply:
But why would you ever want to send NM messages to a ped that is not under NM script control?

Reply:
Fair enough then. I guess no one else would want to send NM messages to a ped that is not under NM script control.